ui_framework/src/components/call_out/_call_out.scss

```diff
@@ -0,0 +1,57 @@
+.kuiCallOut {
+  padding: $kuiSizeM $kuiSizeL;
+  border-left: 4px solid $kuiColorPrimary;
+  background-color: rgba($kuiColorPrimary, 0.1); /* 1 */
```

Comment:
Again, this is where I why I think modifiers should use `&.`. Modifiers modify something. Makes perfect sense to use CSS this way.